treesit-fold.el

```diff
@@ -9,7 +9,7 @@
 ;;         Shen, Jen-Chieh <jcs090218@gmail.com>
 ;; URL: https://github.com/emacs-tree-sitter/treesit-fold
 ;; Version: 0.1.0
-;; Package-Requires: ((emacs "29.1") (tree-sitter "0.15.1") (s "1.9.0") (fringe-helper "1.0.1"))
+;; Package-Requires: ((emacs "29.1") treesit (s "1.9.0") (fringe-helper "1.0.1"))
```

Comment:
We can remove the treesit dependency here.

Reply:
Sure, i've removed it.